docs/notebooks/2.3 Dust Soiling with Kimber.ipynb

```diff
@@ -0,0 +1,398 @@
+      "text/plain": [
+       "<Figure size 1200x500 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "soiling = kimber(\n",
+    "    rainfall_data.precipitation_rate, \n",
+    "    cleaning_threshold=10.0, \n",
+    "    grace_period=3, \n",
+    "    soiling_loss_rate=0.0015\n",
+    ")\n",
+    "\n",
+    "df = pd.concat([rainfall_data, soiling], axis=1)\n",
+    "\n",
+    "f = plot(df.index, df.precipitation_rate, df.accumulated_rainfall, df.soiling, threshold=10)\n",
+    "\n",
+    "print(f\"average soiling loss: {df.soiling.mean()}\") "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8415b306-64db-4115-88e4-92c25abde8c9",
+   "metadata": {},
+   "source": [
+    "The Kimber implementation in PVLib allows to specify `manual_wash_dates`. Below we set a biweekly schedule when our panels will be cleaned, bringing the soling loss back to zero without starting a \"grace period\":"
```

Comment:
End paragraph with full stop.  Lowercase "pvlib".

"The Kimber implementation in pvlib allows specification of `manual_wash_dates`. Below we set a biweekly schedule on which our panels will be cleaned,..." reads a bit better.

Typo "soling"